roxie/udplib/udptrr.cpp

```diff
@@ -1538,7 +1536,7 @@ class CReceiveManager : implements IReceiveManager, public CInterface
         //any other sender for the udpPermitTimeout period
         unsigned maxSlotsPerClient = (udpMaxPendingPermits == 1) ? queue_size : (udpMaxClientPercent * queue_size) / (udpMaxPendingPermits * 100);
         assertex(maxSlotsPerClient != 0);
-        if (maxSlotsPerClient > queue_size)
+        if ((int) maxSlotsPerClient > queue_size)
```

Comment:
Could argue that changing type of queue_size would be preferable but I didn't want to go down that rabbit hole